osclib/check_duplicate_binaries_command.py

```diff
@@ -13,30 +16,41 @@ def __init__(self, api):
     def perform(self, save=False):
         duplicates = {}
         for arch in sorted(target_archs(self.api.apiurl, self.api.project), reverse=True):
-            print('arch {}'.format(arch))
+	    url = self.api.makeurl(['build', self.api.project, 'standard', arch], { 'view': 'binaryversions' })
+	    try:
+		data = http_GET(url)
+	    except urllib2.HTTPError:
+		return
```

Comment:
Any reason to bother catching exception since this will just hide the error and stop immediately? Would seem preferable to just let the exception free.